packages/treasury/test/vault-contract-wrapper.js

```diff
@@ -39,11 +39,17 @@ export async function start(zcf) {
     },
   };
 
-  function stageReward(amount, _fromSeat) {
-    const priorReward = stableCoinSeat.getAmountAllocated('RUN', runBrand);
-    return stableCoinSeat.stage({
-      RUN: AmountMath.add(priorReward, amount),
-    });
+  function reallocateReward(amount, fromSeat, otherSeat) {
+    stableCoinSeat.incrementBy(
+      fromSeat.decrementBy({
+        RUN: amount,
+      }),
+    );
+    if (otherSeat !== undefined) {
+      zcf.reallocate(stableCoinSeat, fromSeat, otherSeat);
+    } else {
+      zcf.reallocate(stableCoinSeat, fromSeat);
+    }
```

Comment:
I think this is a JS style question rather than a request for a change.

I thought javascript would treat `zcf.reallocate(stableCoinSeat, fromSeat, otherSeat);` the same as `zcf.reallocate(stableCoinSeat, fromSeat);` if `otherSeat` is undefined. Do you think the 5-line conditional makes it clearer than a one-liner?

Reply:
Ah, I think that's a misunderstanding. In JavaScript, passing `undefined` as an argument is very different than not passing an additional argument. 

I added a test at the end of packages/zoe/test/unitTests/zcf/test-reallocate-empty.js to demonstrate this. If `undefined` is passed to `zcf.reallocate`, zcf treats it as a seat and tries to find the staged allocation for it. The call to `reallocate` errors.